mta/tests/migration/controls/test_business_services.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from pytest_check import check
+
+from conftest import generate_description, generate_name
+
+
+@pytest.mark.business
+def test_create_business_service(business_service, stakeholder, get_api, delete_api, update_api):
```

Comment:
Please rename the test name: `test_crud_business_service`

Reply:
I think the file name can be test_business_service.py and the test case name can be test_business_service_crud.

as We don't plan to add any other files for business services.

Reply:
Thanks @sshveta and @mguetta1 !
Kept the file name `test_business_service.py` and renamed test case name to `test_business_service_crud`